storybook/cypress/e2e/label.cy.ts

```diff
@@ -114,10 +97,10 @@ describe("Label", () => {
       .get("@dsoLabelShadow")
       .find("button span.sr-only")
       .should("have.text", `Verwijder: ${defaultLabelText}`)
-      .wait(100)
       .get("@dsoLabel")
       .invoke("text", updatedText)
-      .wait(100)
+      .get("@dsoLabel")
+      .should("have.text", updatedText)
       .get("@dsoLabelShadow")
       .find("button span.sr-only")
       .should("have.text", `Verwijder: ${updatedText}`);
```

Comment:
Deze krijg ik niet slagend

Reply:
Ik weet niet waar het anders gestart moet worden. Het enige wat ik kan bedenken is een `@Watch` op `removable`, maar dat lost het niet op

Reply:
Toch gelukt werkend te krijgen, moet het nog wel even nakijken

Reply:
Ik heb nu het syncen in de `componentDidRender` geplaatst, maar ik weet niet zo goed of dit een nette oplossing is

Reply:
Tsja, lastig idd.. Want daarmee introduceren we een extra render die niets doet.

Daarnaast is Stencil ook kritisch op het zetten van state in post render lifecycle hooks: https://stenciljs.com/docs/component-lifecycle#rendering-state